spec/index.bs

```diff
@@ -1425,6 +1425,8 @@ the device **should** be lost to prevent these changes from being observable.
                 Note: {{GPUDeviceLostInfo/message}} should not disclose unnecessary user/system
                 information and should never be parsed by applications.
         </div>
+    1. Complete any outstanding {{GPUBuffer/mapAsync()}} steps.
+    1. Complete any outstanding {{GPUQueue/onSubmittedWorkDone()}} steps.
```

Comment:
Bit handwavy but I think it's OK. There are cross-references in both directions between here and those steps.